frontend/src/services/services.tsx

```diff
@@ -1,15 +1,32 @@
-import { createContext, FC } from 'react';
-import { isVideoExist } from './youtube'
-import { IApiContext, FCProps } from './types'
+import { createContext, FC, useState } from "react";
+import { isVideoExist } from "./youtube";
+import { IApiContext, FCProps } from "./types";
 
 const ApiContext = createContext<IApiContext | null>(null);
 
-const ApiProvider:FC<FCProps> = ({ children }) => {
-  const apiState:IApiContext = {
+const ApiProvider: FC<FCProps> = ({ children }) => {
+  const apiState: IApiContext = {
     isVideoExist,
   };
 
   return <ApiContext.Provider value={apiState}>{children}</ApiContext.Provider>;
 };
 
-export { ApiContext, ApiProvider };
\ No newline at end of file
+const AuthContext = createContext<any | null>(null);
```

Comment:
Could you create an interface for strongly typed AuthContext, to be used instead of `<any>`? something like `IAuthContext`?